lib/core/base/metadata-function-map.js

```diff
@@ -129,7 +129,6 @@ import noAutoplayAudioEvaluate from '../../checks/media/no-autoplay-audio-evalua
 // rule matches
 import ariaAllowedAttrMatches from '../../rules/aria-allowed-attr-matches';
 import ariaAllowedRoleMatches from '../../rules/aria-allowed-role-matches';
-import ariaFormFieldNameMatches from '../../rules/aria-form-field-name-matches';
```

Comment:
This cannot be a `fix` commit and shouldn't be removed (this would be a breaking change).

Instead, this should be a `feat` commit and the prior `ariaFromFieldNameMatches` should be left as it and only deprecated.

Reply:
After discussing with Wilco: the `id` is left in the code so can still be used. The new function does everything the old function did but in a more efficient way, so it's not considered a breaking change. OK to remove the old code.